### PR TITLE
Web linking improvements

### PR DESCRIPTION
**Pull request by gcotelli**
Improve web linking support

- Rename ZnLink to WebLink
- Add methods for well-known parameters
- Add well-known relation types
- Fix handling of parameters needing quotation marks
- Update documentation on web linking

### CODE REVIEW

File: source/Hyperspace-Model-Tests/WebLinkTest.class.st [hunk 1]
```diff
@@ -0,0 +1,289 @@
+"
+I'm a test case for ZnLink
```
Comment (mtabacman) [inline source/Hyperspace-Model-Tests/WebLinkTest.class.st]:
```suggestion
I'm a test case for WebLink
```

File: docs/IETF.md [hunk 2]
```diff
@@ -36,27 +36,41 @@ a request.
...
-Optionally links allow configuring the relation type using the `rel:` message.
+or send `asWebLink` to a URL:
+
+```smalltalk
+'https://www.google.com' asUrl asWebLink
+```
+
+Optionally links allow configuring parameters. Well known parameters
+are provided as configuration methods:
+
```
Comment (mtabacman) [inline docs/IETF.md:64]: Should we keep the setter structure? Isn't it better to rename them to something like `change...to:`  / `set...to:` / `set...as:`?

File: source/Hyperspace-Model/MultiValueOrderedDictionary.class.st [hunk 3]
```diff
@@ -0,0 +1,40 @@
+"
+I'm like ZnMultiValueDictionary but preserving the insertion order of my keys.
```
Comment (mtabacman) [inline source/Hyperspace-Model/MultiValueOrderedDictionary.class.st:2]: Is the reference to ZnMultiValueDictionary outdated?

File: source/Hyperspace-Model/WebLink.class.st [hunk 4]
```diff
@@ -0,0 +1,503 @@
+"
+I represent a Link Header value.
```
Comment (mtabacman) [inline source/Hyperspace-Model/WebLink.class.st]: The class comment is weird, is it the right one?

**Review by mtabacman — CHANGES_REQUESTED** [review]

File: docs/IETF.md (hunk 2 above)
Comment (fortizpenaloza) [reply docs/IETF.md:64]: Hmmm the idea is to make it explicit that we're changing the object?

**Review by fortizpenaloza — DISMISSED** [review]
I agree with the changes proposed by @mtabacman

File: source/Hyperspace-Model/MultiValueOrderedDictionary.class.st (hunk 3 above)
Comment (gcotelli) [reply source/Hyperspace-Model/MultiValueOrderedDictionary.class.st:2]: No. This class is akin ZnMultiValueDictionary but preserving the insertion order. That way, the tests are deterministic.

**Review by mtabacman — APPROVED** [review]

**Pull request merged**